assets/main.js.liquid

```diff
@@ -81,33 +81,189 @@ Site.Header = {
 };
 
 Site.Product = {
+  minWidth: 720, // minWidth for fixed header layout
+  numberRelatedToPick: 4, // number of related products to show
   init: function() {
     var _this = this;
 
-    if ($('#product-gallery').length) {
-      _this.setGalleryWidth();
+    _this.$container = $('.container'); // used by toggleFixed() on scroll
```

Comment:
This matches 7 items on the page. Not sure if should only be focusing on the main product container?